exampleCourse/questions/demo/workspace/rstudio/question.html

```diff
@@ -1,6 +1,25 @@
 <pl-question-panel>
+
 <markdown>
+----
+
+### Compound R Object
+
+Please create a single R object named `x` containing the following:
+- an integer `ii` with value 42
+- a numeric `nn` with value 3.1415
+- a text variable `cc` with text `hello, world!` including the comma an exclamation
+
 You can press the button below to launch a demo RStudio workspace.
 </markdown>
-  <pl-workspace></pl-workspace>
+
+<pl-workspace></pl-workspace>
+
+<pl-file-editor file-name="student.R" ace-mode="ace/mode/r" source-file-name="demo.R"></pl-file-editor>
```

Comment:
Because we do every question that way, and pick the code up from the editor.

Edits in the workspace are "left as an exercise for the reader" and outside the scope of this PR.

Reply:
🙌 thanks @coatless! Let me get this approved and queued to merge.